packages/@react-types/provider/src/index.d.ts

```diff
@@ -92,9 +92,24 @@ export interface ProviderProps extends ContextProps, DOMProps, StyleProps {
 }
 
 export interface ProviderContext extends ContextProps {
+  /**
+   * The package version number of the nearest parent Provider.
+   */
   version: string,
+  /**
+   * The theme of the nearest parent Provider.
```

Comment:
Do we need to mention any default values?

Reply:
Well, the values trickle down from the parent providers if not explicitly defined in the nested one and IMO I think the prop default definitions will be sufficient then